diff_script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+./a.out units/halis.json units/codos.json > fight.txt
+./a.out units/halis.json units/kowa.json >> fight.txt
+./a.out units/codos.json units/halis.json >> fight.txt
+./a.out units/codos.json units/kowa.json >> fight.txt
+./a.out units/kowa.json units/halis.json >> fight.txt
+./a.out units/kowa.json units/codos.json >> fight.txt
+
+DIFF=$(diff fight.txt manual_calculation.txt)
```

Comment:
itt megcsinalod ezt a DIFF-et, aztan a workflow-ban is meghivod meg egyszer. Igy redundansan ne legyen. Terjen vissza akkor ez a fuggveny a DIFF eredmenyevel, es ott ne legyen emg egyszer lefuttatva a diff.

Reply:
MArmint a script terjen vissza 0-val ha a DIFF ures, es hibakoddal ha nem.